save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -5,137 +5,407 @@
+                                }
+                            }
+                        }
+                    }
+
+                    // Collapse card to load binary file
+                    div("card shadow mb-4") {
+                        div("card-header") {
+                            attrs.id = "headingSecond"
+                            button(classes = "btn btn-link collapsed") {
+                                attrs["data-toggle"] = "collapse"
+                                attrs["data-target"] = "#collapseSecond"
+                                attrs["aria-expanded"] = "false"
+                                attrs["aria-controls"] = "collapseSecond"
+                                h6("m-0 font-weight-bold text-primary") {
+                                    +"Upload project as binary file"
+                                }
+                                attrs.onClickFunction = {
+                                    if (numberOpenningCard == 2) {
+                                        numberOpenningCard = 0
+                                    } else {
+                                        numberOpenningCard = 2
+                                    }
+                                }
+                            }
                         }
-                        input(type = InputType.text, name = "itemText") {
-                            attrs {
-                                placeholder = "save.properties"
-                                onChangeFunction = {
-                                    val target = it.target as HTMLInputElement
-                                    pathToPropertyFile = target.value
+                        div("collapse") {
+                            attrs.id = "collapseSecond"
+                            attrs["aria-labelledby"] = "headingSecond"
+                            attrs["data-parent"] = "#accordion"
+                            div("card-body") {
+                                div("mb-3") {
+                                    h6(classes = "d-inline mr-3") {
+                                        +"Binary file: "
+                                    }
+                                    div {
+                                        label {
+                                            input(type = InputType.file) {
+                                                attrs.hidden = true
+                                                attrs {
+                                                    onChangeFunction = { event ->
+                                                        val target = event.target as HTMLInputElement
+                                                        setState { binaryFile = target.files?.let { it[0] } }
+                                                    }
+                                                }
+                                            }
+                                            strong { +"Choose binary file:" }
```

Comment:
'Upload' or 'Attach' would be more appropriate, also some sort of icon would make it more clear